e2e/helper/hotkeys/hotkeys.js

```diff
@@ -0,0 +1,31 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2024, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import { copy, paste, selectAll } from './clipboard.js';
+
+const hotkeys = {};
+
+hotkeys.copy = copy;
+hotkeys.paste = paste;
+hotkeys.selectAll = selectAll;
+
+export default hotkeys;
```

Comment:
good change. i did this whole clipboard to hotkeys thing because i saw some openmct hotkeys being used in tests as well. but turns out they are used slightly differently so i ignored those for now.